src/stories/Library/search-result-page/SearchResultTitle.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { withDesign } from "storybook-addon-designs";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { SearchResultTitle } from "./SearchResultTitle";
+
+export default {
+  title: "Blocks / Search Result Header Title",
+  component: SearchResultTitle,
+  decorators: [withDesign],
+  argTypes: {
+    title: {
+      control: { type: "text" },
+      defaultValue: "harry potter",
+    },
+    totalResults: {
+      control: { type: "number" },
+      defaultValue: 100,
+    },
+    zeroResult: {
+      description: "Do we have zero results?",
+      control: { type: "boolean" },
+      defaultValue: false,
+    },
```

Comment:
Please consider if we really need the `zeroResult` prop/component.

With the new `isLoading` prop it seems as if a `totalResults` value of `0` and `isLoading` value of `false` that we will know that this is in fact a `zeroResult`.

Reply:
I think you are making the component/story too smart. This is only to represent the markup. Maybe you can combine the params by clicking the controls in SB and end in a state that is not possible, but then it is your own fault.